core/fileutil/dir_windows.go

```diff
@@ -0,0 +1,10 @@
+package fileutil
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func DefaultDir() string {
+	return filepath.Join(os.Getenv("LOCALAPPDATA"), "Chain")
```

Comment:
We can totally just call this directory Chain Core--nothing else is here at the moment. (Postgres should probably put its data files here too, but it doesn't, yet.)